src/box/lua/schema.lua

```diff
@@ -2340,8 +2340,21 @@ end
 
 box.internal.check_select_opts = check_select_opts -- for net.box
 
+local check_select_args_rl = log.internal.ratelimit:new()
+local function check_select_args(index, key, opts)
+    local rl = check_select_args_rl
```

Comment:
`logger` would be better, I think.

Reply:
It were a simple logger, I would simply use the `log` module API: in terms of OOP, `Ratelimit` would actually be a subclass of `Logger` (if we had one) — I want to emphasize its semantics (of rate limiting log messages).